stackprinter/formatting.py

```diff
@@ -195,7 +195,7 @@ def format_exc_info(etype, evalue, tb, style='plaintext', add_summary='auto',
                                             exc,
                                             exc.__traceback__,
                                             chain=False)
-        where = ', '.join(str(w) for w in getattr(exc, 'where', []))
+        where = getattr(exc, 'where', None)
```

Comment:
```suggestion
        where = getattr(exc, 'where', '')
```

Then remove the `if` below.

Reply:
the `if` below switches off the whole subclause (`" while formatting "`)

Reply:
oops, nvm